### DISCUSSION TITLE
Keyboard shorcut `gd` now opens the "Discussions" tabs whereas previously it went to Dashboard

### DISCUSSION THREAD

**Discussion by ebouchut** [community/community#2998] [Discussions] [answered]
Hi,

First off "Discussions" is a nice and useful addition,
and providing a keyboard shortcut `gd` for this purpose is a must, thanks.

However  it  previously opened our _Dashboard_ whereas now opens the _Discussions_ tab.
Hence both stamps on each other toes at least during the beta to make things faster for everyone.

This is just a reminder for you to instead keep in mind to choose another keyboard shortcut that leads to the _Discussions_ tab.

**Comment by becca** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/2906, we're working to resolve this bug.

**Reply by becca** [reply]
This issue should now be fixed. We'll be updating the documentation shortly but wanted to let you know that `gd` lands you on the dashboard and we'll be adding `gg` for Discussions
